app/code/Magento/Cms/view/adminhtml/web/css/source/_module.less

```diff
@@ -3,13 +3,20 @@
  * See COPYING.txt for license details.
  */
 .modal-slide {
-  .media-gallery-modal {
-    .page-main-actions {
-      margin-bottom: 3rem;
+    .media-gallery-modal {
+        .page-main-actions {
+            margin-bottom: 3rem;
+        }
+
+        #new_folder {
```

Comment:
Please avoid using id selectors
You can read more [here](https://devdocs.magento.com/guides/v2.3/coding-standards/code-standard-less.html#selectors)

Reply:
Hi @Stepa4man :
Thank you for your reviewing. But in my previous comment, the static test is failed but it is in the old source code and currently, I don't find the best way to avoid the id selectors.

Reply:
Hey, I know that some of the Magento legacy code is doesn't follow the rules. Unfortunately, there are a lot of technical debts. But it doesn't mean that we don't have to fix it during some changes.
Did you consider to add a new CSS class to the HTML node and declare a new class in the LESS file?

Reply:
Hi @Stepa4man,

Reply:
Hi @Stepa4man :

I have changed the source code. Please check it again.

Thank you :)

Reply:
Please check static test fail one more time, there is some more legacy that can be fixed.
Thanks in advance.

Reply:
Hi @Stepa4man :
It should be OK now :) Please check it again.
Thanks!